### PR TITLE
German paths & error page

### PR DESCRIPTION
**Pull request by tpreusse**
Routes diff:
https://github.com/orbiting/republik-frontend/pull/49/files#diff-52503f2920dcbf5082edfc5670ede0f1

Redirect for old ones are prepared and in our secure shared file system.

Redirect example: https://republik.love/merci

Example: https://republik.love/2017/11/28/meta

screen shot 2018-01-10 at 03 39 59

### CODE REVIEW

File: lib/routes.js [hunk 1]
```diff
@@ -4,26 +4,26 @@ const routes = nextRoutes()
...
+  .add('media', '/medien')
+  .add('pledge', '/angebote')
+  .add('claim', '/abholen')
   .add('updates', '/updates/:slug?')
-  .add('notifications', '/notifications')
-  .add('legal/imprint', '/legal/imprint')
-  .add('legal/privacy', '/legal/privacy')
-  .add('legal/statute', '/legal/statute')
-  .add('legal/tos', '/legal/tos')
+  .add('notifications', '/notifikation')
```
Comment (clarajeanne) [inline lib/routes.js]: can we call that 'benachrichtigungen'?

File: components/Error/index.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+const getRedirect = gql`
+query getRedirect($path: String!) {
+  redirection(path: $path) {
+    target
+    status
+  }
+}
+`
+
+const ErrorComponent = ({statusCode, t, loading, children}) => (
```
Comment (uxengine) [inline components/Error/index.js]: Maybe name this something like StatusError? ErrorComponent could be all kinds of things so a more specific name could be helpful moving forward.

File: lib/routes.js (hunk 1 above)
Comment (uxengine) [inline lib/routes.js]: or maybe just `/meldung `to keep it short?

Comment (tpreusse) [reply lib/routes.js]: I like to word but consider it too newsy to use here—«eine Republik Meldung» should just be a sign in confirmation^^

Comment (uxengine) [reply lib/routes.js]: maybe /mitteilung then, like the page title?

Comment (tpreusse) [reply lib/routes.js]: mitteilung it is

File: components/Error/Frame.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            {children}
+          </NarrowContainer>
+        </div>
+      </Fragment>
+    )
+  }
+}
+
+const y = (scroll, pos) => {
+  return (pos < scroll ? pos : scroll) * -0.55
```
[the remark this answers is not in the document]
Comment (tpreusse) [reply components/Error/Frame.js]: Agree but going to skip it for now.

**Pull request merged**